README.asciidoc

```diff
@@ -0,0 +1,401 @@
+# picr22-team-valdis
+
+Team members
+
+*Electronics:* Karl Sander Vinkel
+
+*Mechanics:* Karl Sander Vinkel
+
+*Programming:* Kevin Kirbits, Markus Männil, Arnold Leonhard Mertsin
+
+
+== Project Links
+
+https://a360.co/3goN0VZ[Valdis Fusion360]
+
+
+== Info
+
+Anydesk ip 198414610 - Valdis111
+
+SSH valdis@valdis password: valdis
+
+static ip: 172.17.152.9
+
+== Programming
+
+Libraries: 
+
+State Machine: 4 main states (FIND_BALL, MOVE_TO_BALL, ORBIT, THROW) FIND -> MOVE -> ORBIT -> THROW. Every tick robot decides in which state it's in from image processing
+
+opencv2: used for image processing
```

Comment:
Did not find a more detailed description as how to set up the robots code, (compiling the segmentation module, for example) and configure the robot. when that is added to the software/README file, it should be in its pull request